src/Apps/Order/Routes/Shipping2/Shipping2.jest.tsx

```diff
@@ -781,13 +794,15 @@ describe("Shipping", () => {
               recommendedAddress
             )
 
-            mockCommitMutation.mockResolvedValueOnce(
-              settingOrderShipmentSuccess
-            )
+            mockCommitMutation
+              // .mockResolvedValueOnce(settingOrderShipmentSuccess)
+              .mockResolvedValueOnce(updateAddressSuccess)
             // Clicking "Use This Address" on verification modal automatically
             // sets shipping on the order and proceeds to the next step.
-            userEvent.click(screen.getByText("Use This Address"))
-            expect(mockCommitMutation).toHaveBeenCalledTimes(1)
+            await userEvent.click(screen.getByText("Use This Address"))
+            await waitFor(() => {
+              expect(mockCommitMutation).toHaveBeenCalledTimes(2)
+            })
```

Comment:
It's possible this assertion had to change because the system is better at waiting now - the extra request is due to the user address being saved.